### ISSUE TITLE
Imported API with wrong examples

### ISSUE THREAD

**Issue by DMaxter** [a85/POSTMan-Chrome-Extension#11924]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

When importing an external API, the generated collection/API doesn't have the right examples generated in the body. Tried with many APIs and all generate the same way. This used to work before.

### Steps To Reproduce

1. Click `Import` on the top bar
2. Copy a URL or the API spec (https://api.apis.guru/v2/specs/1password.com/events/1.0.0/openapi.json for example)
3. Any option can be selected and then click `Import`
4. Go to a request with a POST (so it will have a body) - if using the 1password I gave as example, it is in the `/api/v1/signinattempts` folder
5. Check the generated example

### Screenshots or Videos

Generated this:
![image](https://user-images.githubusercontent.com/35278584/232107404-278fb574-b71e-4ae4-b493-dd7101ed34b8.png)
When something like this should be generated (got it from the examples)
![image](https://user-images.githubusercontent.com/35278584/232107913-b3d37756-78c3-439e-84d9-fddc9762609e.png)

### Operating System

Windows

### Postman Version

10.12.13

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by VShingala** [comment]
@DMaxter While importing this OpenAPI, you can select the behaviour of how the requests should be generated. You can choose options via `View Import Settings` button at bottom right of import modal.

![image](https://user-images.githubusercontent.com/16756296/232117529-44ec2e48-adaa-44a5-90b4-7c1856af7a83.png)

Further, you can choose `Example` as a value for the option named `Parameter generation`. This means, while generating data from OpenAPI definition, we'll try to use examples defined in OpenAPI definition.

![Screenshot 2023-04-14 at 11 08 42 PM](https://user-images.githubusercontent.com/16756296/232118165-018d08c3-1f21-4b6d-8513-ca6535e271ef.png)

I've tried this and I was able to get correct `cursor` value suggested by you. i.e. Same as what's mentioned in Definition.

**Comment by DMaxter** [comment]
But before I never had to do this. Did the default behavior change?

**Comment by VShingala** [comment]
@DMaxter We recently had a change to options that defined this behaviour. Earlier we used to have 2 options with same behavior for request and examples parts of collection separately namely `Request parameter generation` and `Response parameter generation`. Default option value for these options were different, `Schema` and `Example` respectively. We've recently combined this into one option, for which the default value is now `Schema` and thus the change in behavior.
